<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLD_SetPoint</class>
 <widget class="QDialog" name="WLD_SetPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selecting point on the world map</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/editor.qrc">
    <normaloff>:/toolbar/dock/world.png</normaloff>:/toolbar/dock/world.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3" rowspan="2" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7" rowspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="8">
    <widget class="GraphicsWorkspace" name="graphicsView">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="7">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Note: Use the mousewheel to scroll on the map. You can scroll horizontally by holding down the CTRL key.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current Point:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="GotoPoint">
     <property name="toolTip">
      <string>You will change current position to the last selected point. If point is not set, position will not be changed.</string>
     </property>
     <property name="text">
      <string>Go to point</string>
     </property>
     <property name="icon">
      <iconset resource="../../_resources/editor.qrc">
       <normaloff>:/toolbar/dock/search.png</normaloff>:/toolbar/dock/search.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="animation">
     <property name="toolTip">
      <string>Start and stop animation of placed on the map items.</string>
     </property>
     <property name="text">
      <string>Animation</string>
     </property>
     <property name="icon">
      <iconset resource="../../_resources/editor.qrc">
       <normaloff>:/images/animation.png</normaloff>:/images/animation.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="ResetPosition">
     <property name="toolTip">
      <string>You will return to the x0-y0 position of the map.</string>
     </property>
     <property name="text">
      <string>Reset position</string>
     </property>
     <property name="icon">
      <iconset resource="../../_resources/editor.qrc">
       <normaloff>:/images/reset_pos.png</normaloff>:/images/reset_pos.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">X:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">Y:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="x_point">
     <property name="text">
      <string notr="true">[Empty]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="y_point">
     <property name="text">
      <string notr="true">[Empty]</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsWorkspace</class>
   <extends>QGraphicsView</extends>
   <header>common_features/graphicsworkspace.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>graphicsView</tabstop>
  <tabstop>ResetPosition</tabstop>
  <tabstop>GotoPoint</tabstop>
  <tabstop>animation</tabstop>
 </tabstops>
 <resources>
  <include location="../../_resources/editor.qrc"/>
 </resources>
 <connections/>
</ui>
